ext/soap/soap.c

```diff
@@ -4021,7 +4021,7 @@ static xmlNodePtr serialize_parameter(sdlParamPtr param, zval *param_val, int in
 	} else {
 		if (name == NULL) {
 			paramName = paramNameBuf;
-			snprintf(paramName, sizeof(paramNameBuf), "param%d",index);
+			snprintf(paramNameBuf, sizeof(paramNameBuf), "param%d",index);
```

Comment:
Isn't that a const violation? (not that I know C's const rule that well)

Reply:
I don't think so but I swapped the lines anyway.